entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Getting Started - EF Core
+author: masoucou
+ms.date: 07/07/2020
+uid: core/get-started/xamarin
+---
+
+# Getting Started with EF Core and Xamarin
+
+In this tutorial, you create a Xamarin.Forms application that performs data access against a SQLite database using Entity Framework Core.
+
+You can follow the tutorial by using Visual Studio on Windows or Visual Studio for Mac.
+
+[View this article's sample on GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/GetStarted).
+
+## Prerequisites
+
+Install one of the below:
+
+* [Visual Studio 2019 version 16.3 or later](https://www.visualstudio.com/downloads/) with this  workload:
+  * **Mobile Development with .NET**
+* [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/)
+
+This [documentation provides detailed step-by-step installation instructions](https://docs.microsoft.com/xamarin/get-started/installation) for each platform.
+
+## Download and run the sample project
+
+To run and explore this sample application, download the code on GitHub.
+
+Once downloaded, open the solution file `EFGettingStarted.sln` in Visual Studio or Visual Studio for Mac and run the application on the platform of your choice.
+
+When the app first starts, it will populate the local SQLite database with 2 entries representing blogs.
+
+Click the **Add** button in the toolbar.
+
+A new page will appear that allows you to enter information about a new blog. 
+
+Fill out all the info and click **Save** from the toolbar. The new blog will save to the app's SQLite database and will show in the list.
+
+You can click on one of the blog entries in the list and see any posts for that blog.
+
+Click **Add** in the toolbar.
+
+A page then appears that allows you to fill out information about a new blog post. Fill out all the information and click **Save** in the toolbar.
+
+The new post will be associated to the blog post you clicked on in a previous step and will save to the app's SQLite database and show in the list.
+
+Go back to the blog list page. And click **Delete All** in the toolbar. All blogs and their corresponding posts will then be deleted from the app's SQLite database.
+
+## Explore the code
+
+The following sections will walk you through the code in the sample project that reads, creates, updates, and deletes data from a SQLite database using EF Core with Xamarin.Forms.
+
+It is assumed that you are familiar with the Xamarin.Forms topics of displaying data and navigating between pages.
+
+## Entity Framework Core NuGet packages
+
+To create Xamarin.Forms apps with EF Core, you install the package for the EF Core database provider(s) you want to target into all of the projects in the Xamarin.Forms solution. This tutorial uses SQLite.
+
+The following NuGet packages are needed in each of the projects in the Xamarin.Forms solution.
+
+* `Microsoft.EntityFrameworkCore`
```

Comment:
Note: you only have to install the Sqlite package. It will pull in the core package.

Reply:
D'oh.